common.go

```diff
@@ -17,3 +18,20 @@ func debugLog(format string, args ...interface{}) {
 	}
 	log.Printf(format+"\n", args...)
 }
+
+//logURL replaces existing password from the given URL
+func logURL(URL string) (string, error) {
+	// Hide password if set in URL
+	u, err := url.Parse(URL)
+	if err != nil {
+		return "", err
+	}
+	if u.User != nil {
+		_, exists := u.User.Password()
+		if exists {
+			u.User = url.UserPassword(u.User.Username(),
+				"__password__")
```

Comment:
Don't wrap this line.

https://github.com/golang/go/wiki/CodeReviewComments#line-length